diff --git a/llvm-build/build.py b/llvm-build/build.py
index d63696876939..02a4b168b457 100755
--- a/llvm-build/build.py
+++ b/llvm-build/build.py
@@ -920,7 +920,7 @@ class LlvmCore(BuildUtils):
 
             if self.build_config.build_libedit:
                 llvm_defines['LibEdit_LIBRARIES'] = \
-                    self.merge_install_dir('libedit', self.use_platform(), 'lib', 'libedit.so.0.0.76')
+                    self.merge_install_dir('libedit', self.use_platform(), 'lib', 'libedit.so')
 
             if not build_instrumented and not no_lto and not debug_build:
                 llvm_defines['LLVM_ENABLE_LTO'] = 'Thin'
